Aspire Budgeting/AppDelegate.swift

```diff
@@ -3,10 +3,24 @@
 //  Aspire Budgeting
 //
 
+import SwiftyBeaver
 import UIKit
 
+let Logger = SwiftyBeaver.self
+
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/8201815ecbc80b5aecdffa5b8b2e54cd2b50ebe4/Aspire Budgeting/AppDelegate.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>